visualdl/server/lib.py

```diff
@@ -552,3 +553,84 @@ def _handler(key, func, *args, **kwargs):
         return data
 
     return _handler
+
+
+def get_network(log_reader, stage):
+    response = {}
+    network_dir = log_reader.work_dir + "/output/" + stage + "_network.json"
```

Comment:
```suggestion
    network_dir = os.path.join(log_reader.work_dir, "output", stage, "_network.json")
    network_dir = os.path.normpath(network_dir)
```
避免在不同平台上出现`\\`和`/`混用的问题